lib/state/action/pokemon_actions.dart

```diff
@@ -1,12 +1,17 @@
 import 'dart:async';
 
+import 'package:async_redux/async_redux.dart';
 import 'package:pokedex_async_redux/api/api_service.dart';
+import 'package:pokedex_async_redux/state/action/actions.dart';
 import 'package:pokedex_async_redux/state/app_state.dart';
 import 'package:pokedex_async_redux/utilities/constant.dart';
-import 'package:async_redux/async_redux.dart';
 
 /// Getting of Pokemons from Pokemon API
-class GetPokemonsAction extends ReduxAction<AppState> {
+class GetPokemonsAction extends LoadingAction {
+  static const key = getPokemonActionKey;
```

Comment:
this is already a constant right? therefore no need to create a separate one for this 😉